lib/active_model_serializers/key_transform.rb

```diff
@@ -66,5 +66,28 @@ def underscore(value)
     def unaltered(value)
       value
     end
+
+    # Transforms string to selected case
+    # Accepts string in any case: 'camel', 'undescore', 'dashed'.
+    #
+    # @example:
+    #   transform('SomeClass', :underscore) => 'some_class'
+    #   transform('some_class', :snake) => 'someClass'
+    #   etc...
+    def transform(string, type)
```

Comment:
When I originally wrote this module it did not include the transform method. https://github.com/rails-api/active_model_serializers/blob/3498647d1a07b9e8a183702bef53cf1ce8a76370/lib/active_model_serializers/key_transform.rb

It seems to have appeared in https://github.com/rails-api/active_model_serializers/commit/913f396bb17823bf88f57afbb1abcee8bad60f32